app/assets/javascripts/app/discussions/templates/desktop.html.slim

```diff
@@ -22,4 +23,4 @@ TopBarLayout
         label Tags
         input ng-model="ctrl.discussion.tags" type="text"
 
-      textarea.Editor placeholder="Write the fundaments of the discussion" ng-model="ctrl.discussion.body" required=""
+      textarea.Editor.Editor-noSidePadding placeholder="Write the fundaments of the discussion" ng-model="ctrl.discussion.body" required=""
```

Comment:
That would be the right usage of SUIT. But the name is still bad, it should not be couple to the rules.